services/post_service.py

```
@@ -0,0 +1,59 @@
+
+    @staticmethod
+    def _verify_author(post:dict, current_user_id: str):
+        if post["author_id"] != current_user_id:
+            raise HTTPException(status_code=status.HTTP_403_FORBIDDEN,
+                                detail="해당 게시글에 대한 수정/삭제 권한이 없습니다.")
+
+    async def create_post(self,req:PostsCreateRequest, author_id : str) -> dict:
+        post_data = req.model_dump()
+
+        post_data["author_id"] = author_id
+        post_data["created_at"] = datetime.now()
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)